src/configs/esbuild.es5.config.js

```diff
@@ -101,7 +103,7 @@ module.exports = (folder, globalName) => {
     target: 'es5',
     mainFields: buildHelpers.getResolveConfigForBundlers(),
     outfile: `${folder}/appBundle.es5.js`,
-    sourcemap,
+    sourcemap: sourcemap,
```

Comment:
is this necessary?

Reply:
As per the documentation, it has to be like that. If we want we can revert this to original

Reply:
it should be the same according to JS syntax. And the original version is a bit shorter and cleaner, I think 😊